htdocs/kernel/blockinstance.php

```diff
@@ -63,7 +63,7 @@ public function __set($name, $args)
     {
         trigger_error("Class '" . __CLASS__ . "' is deprecated" . (empty($name) ? '' : " thus the variable '{$name}' is not set") . '!', E_USER_WARNING);
 
-        return false;
+        return null;
```

Comment:
What would you think about just deleting this? It has been this way since 2.2.

Reply:
I'm not aware of any module still using it, so it makes sense to delete it.